iotdb-core/datanode/src/main/java/org/apache/iotdb/db/pipe/extractor/dataregion/historical/PipeHistoricalDataRegionTsFileAndDeletionExtractor.java

```diff
@@ -251,7 +258,8 @@ public void customize(
       final PipeParameters parameters, final PipeExtractorRuntimeConfiguration configuration)
       throws IllegalPathException {
     shouldExtractInsertion = listeningOptionPair.getLeft();
-    // Do nothing if only extract deletion
+    shouldExtractDeletion = listeningOptionPair.getRight();
+    // Do nothing if extract deletion
```

Comment:
If only extract deletion? Now for the user pipes only-extracting deletion, the persisted deletion can be used too. However this may cause extra deletion if the sender pipes restart too late and the receiver has some data locally written to it...

Reply:
Well, it's better to enable the historical extractor when only extracts deletion now. However, it can be checked later in other PR.

Reply:
TODO with @Caideyipi